src/parameterizations/vertical/MOM_geothermal.F90

```diff
@@ -413,10 +419,10 @@ subroutine geothermal_init(Time, G, param_file, diag, CS)
   if (id > 0) call post_data(id, CS%geo_heat, diag, .true.)
 
   ! Diagnostic for tendency due to internal heat (in 3d)
-  CS%id_internal_heat_tend_3d = register_diag_field('ocean_model',&
-        'internal_heat_tend_3d', diag%axesTL, Time,             &
-        'Internal heat tendency in 3D, reveals layer(s) that heat is added to','W m-2',&
-        v_extensive = .true.)
+  CS%id_internal_heat_tend_3d=register_diag_field('ocean_model', &
+        'internal_heat_tend_3d', diag%axesTL, Time,              &
+        'Internal heat tendency in 3D, reveals layer(s) that heat is added to', &
+        'W m-2', v_extensive = .true.)
   
```

Comment:
The whitespace in this line needs to be removed.

Reply:
Sorry, I couldn't seem to find the whitespace that you're referring to...?

Reply:
[From the Travis report](https://travis-ci.org/NOAA-GFDL/MOM6/jobs/575596043):
```
$ ./.testing/trailer.py -e TEOS10 -l 120 src config_src

src/parameterizations/vertical/MOM_geothermal.F90, line 458: Blank line contains spaces

The command "./.testing/trailer.py -e TEOS10 -l 120 src config_src" exited with 1.
```
This is why the PR is still showing a red X.

Reply:
BTW it looks like this whitespace is not showing in GitHub.

Reply:
I think I might have caught this in the last commit. Is it still failing?

Reply:
OK, I see that it is. Will try to get this sorted.